statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -1113,6 +1113,199 @@ def zvalues(self):
         """
         return self.params / self.bse
 
+    def test_normality(self):
+        """
+        Jarque-Bera test for normaility of standardized residuals.
+
+        Null hypothesis is normality.
+
+        Notes
+        -----
+        If the first `d` loglikelihood values were burned (i.e. in the
+        specified model, `loglikelihood_burn=d`), then this test is calculated
+        ignoring the first `d` residuals.
+
+        See Also
+        --------
+        statsmodels.stats.stattools.jarque_bera
+
+        """
+        from statsmodels.stats.stattools import jarque_bera
+        d = self.loglikelihood_burn
+        return np.array(jarque_bera(
+            self.filter_results.standardized_forecasts_error[:, d:],
+            axis=1
+        )).transpose()
```

Comment:
I like the `method` idea, and I like the forced selection. Gives more options for future upgrade paths hopefully. I will add a commit, please let me know if it matches the interface you're thinking of.